proto/frequenz/dispatch/v1/dispatch.proto

```diff
@@ -75,6 +75,62 @@ message Dispatch {
 
   // The dispatch payload
   google.protobuf.Struct payload = 11;
+
+  // The recurrence rule
+  RecurrenceRule recurrence_rule = 12;
+}
+
+// Information about the schedule for reoccurring dispatches
+//
+// Example:
+// Run at noon every day: RecurrenceRule {
+//   seconds = [0],
+//   minutes = [0],
+//   hours = [12],
+//   day_of_month = [],
+//   day_of_week = [],
+//   month = [],
+//   year = [],
+//   count = 0,
+//   // Do not set the `until` field
```

Comment:
So if I add `day_of_week = [1]`, then it will reoccur every Monday at 12?
But If i also add month = [1], then it would only reoccur on every Monday in January every year?
To me it reads like if the next bigger timestep is defined then it's considered, otherwise not. If this is the case, then this should be specifically documented.